### PR TITLE
DSPDC-667 DSPDC-895 Initialize transformation and schema

### PR DESCRIPTION
**Pull request by danxmoran**
Lots of code (but it's mostly boilerplate)

### CODE REVIEW

File: schema/src/main/jade-tables/hles_owner.table.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "od_race_other",
+      "datatype": "string"
+    },
+    {
+      "name": "od_hispanic",
+      "datatype": "string"
```
Comment (danxmoran) [inline schema/src/main/jade-tables/hles_owner.table.json:32]: This should be a boolean

File: schema/src/main/jade-tables/hles_owner.table.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      "datatype": "boolean"
+    },
+    {
+      "name": "od_annual_income_range_usd",
+      "datatype": "string"
+    },
+    {
+      "name": "oc_household_adult_count",
+      "datatype": "string"
+    },
```
Comment (snwessel) [inline schema/src/main/jade-tables/hles_owner.table.json:41]: @danxmoran in the mapping doc, there is also an `oc_household_person_count` column. You left that out intentionally, right?

Comment (danxmoran) [reply schema/src/main/jade-tables/hles_owner.table.json:41]: Nope, good catch

Comment (danxmoran) [reply schema/src/main/jade-tables/hles_owner.table.json:41]: Can you add it and push to the branch? Or fix it when working on the mapping implementation? Deep in the WIP weeds on the dog transformations

**Pull request merged**